src/shims/intrinsics.rs

```diff
@@ -75,40 +75,35 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 this.copy_op(args[1], place.into())?;
             }
 
-            "atomic_load" |
-            "atomic_load_relaxed" |
-            "atomic_load_acq" => {
+            "atomic_load" | "atomic_load_relaxed" | "atomic_load_acq" => {
```

Comment:
We usually keep these one-per-line.

Reply:
There's no way to tell rustfmt about this

Reply:
Should I skip these `match`es?

Reply:
> There's no way to tell rustfmt about this

Well, bad for rustfmt then I'd say. Or do you not agree with me that one-per-line is preferable for our long matches here?

Reply:
I agree that we should have one per line. I have seen rustfmt do this for a few, but not for others. I'll investigate